src/cargo/util/toml/mod.rs

```diff
@@ -669,10 +669,16 @@ fn normalize_package_toml<'a>(
         .map(|value| field_inherit_with(value, "authors", || inherit()?.authors()))
         .transpose()?
         .map(manifest::InheritableField::Value);
+    // Check if multiple build scripts are allowed or not
+    let multiple_build_scripts = gctx.cli_unstable().multiple_build_scripts;
     let build = if is_embedded {
         Some(TomlPackageBuild::Auto(false))
     } else {
-        targets::normalize_build(original_package.build.as_ref(), package_root)
+        targets::normalize_build(
```

Comment:
"the hackiest hack is the best hack"
or
"when implementing short-lived or hacky code, be as surgical as possible"

I'd recommend we do the unstable check here, rather than updating `normalize_build`s API to take in a parameter related to this

Reply:
Should be done